README.md

```diff
@@ -0,0 +1,11 @@
+# Youst
+
+> The player controls a **yellow knight** riding a flying ostrich or stork, from a third-person perspective.
+
+Our goal is to make an application that will be our playground. We want to show our workflow and our best practices through a tool generated by us and for us as we growth as developers.
```

Comment:
I think it should be grow instead of growth

Reply:
Nice catch!